src/librustc_error_codes/error_codes/E0226.md

```diff
@@ -0,0 +1,20 @@
+Only a single explicit lifetime bound is permitted on trait objects.
```

Comment:
This should be the explanation, please move it below the failing code example and instead write:

> More than one explicit lifetime bound was used on a trait object.

Reply:
Thank you ! Is the explanation correct now ?